libfreerdp/codec/h264_mediacodec.c

```diff
@@ -86,13 +87,18 @@ typedef AMediaFormat* (*AMediaCodec_getInputFormat_t)(AMediaCodec*);
 const int COLOR_FormatYUV420Planar = 19;
 const int COLOR_FormatYUV420Flexible = 0x7f420888;
 
+const int MEDIACODEC_MINIMUM_WIDTH = 320;
+const int MEDIACODEC_MINIMUM_HEIGHT = 240;
```

Comment:
As for the `320x240` is there a specific reason for this resolution or is the requirement just `!= 0` ? If the latter, better use `16` or similar as `FreeRDP` is used embedded where displays might be really small (h264 alignment is 16 if I recall correctly?)

Reply:
According to MediaCodec documentation smaller resolutions than 320*240 are not supported:

https://developer.android.com/reference/android/media/MediaCodec#qualityFloor

With Java it's possible to use `MediaCodecInfo.VideoCapabilities` class (https://developer.android.com/reference/android/media/MediaCodecInfo.VideoCapabilities) in order to get the actual minimum/maximum resolutions supported by the codec but unfortunately this API is not exported to the NDK yet

Reply:
Ok, then we´ll just take this. Might be a good idea to reference the link with the constants.